flake.nix

```diff
@@ -19,10 +19,23 @@
         let
           pkgs = nixpkgsFor.${system};
         in {
+          app-autoscaler-cli-plugin = pkgs.buildGoModule rec {
+            name = "app-autoscaler-cli-plugin";
+            src = pkgs.fetchgit {
+              url = "https://github.com/cloudfoundry/app-autoscaler-cli-plugin";
+              rev = "f46dc1ea62c4c7bd426c82f4e2a525b3a3c42300";
+              hash = "sha256-j8IAUhjYjEFvtRbA6o2vA7P2uUmKVYsd9uJmN0WtVCM=";
+              fetchSubmodules = true;
+            };
+            doCheck = false;
+            vendorHash = "sha256-NzEStcOv8ZQsHOA8abLABKy+ZE3/SiYbRD/ZVxo0CEk=";
+          };
```

Comment:
There are some conventions regarding the outputs of a nix-flake, see [NixOS-Wiki-article](<https://nixos.wiki/wiki/Flakes#Output_schema>). To make this package (separately) installable (e.g. via [nix profile install](<https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-profile-install#examples>)), I propose to move it into the `packages`-output.

Additionally: Why don't we move the package-definition into a flake of the [go-plugin-repository](<https://github.com/cloudfoundry/app-autoscaler-cli-plugin>)? It is ours and we (and othersas well) can then consume it from there as an input!

Reply:
> There are some conventions regarding the outputs of a nix-flake, see [NixOS-Wiki-article](https://nixos.wiki/wiki/Flakes#Output_schema). To make this package (separately) installable (e.g. via [nix profile install](https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-profile-install#examples)), I propose to move it into the packages-output.

Okay like that 6414dac?

> Additionally: Why don't we move the package-definition into a flake of the [go-plugin-repository](https://github.com/cloudfoundry/app-autoscaler-cli-plugin)? It is ours and we (and othersas well) can then consume it from there as an input!

I can't oversee the effort that comes with it, especially related to keeping this then up to date for e.g. new releases.

Reply:
Let's do that together if you want. Including explanations, we should be fast enough to do it within ca. 30 minutes.

Reply:
> Let's do that together if you want. Including explanations, we should be fast enough to do it within ca. 30 minutes.

As discussed on the phone, this concern will be adressed in a follow-up PR